spec/features/manage_cmr/subscriptions/create_subscriptions_spec.rb

```diff
@@ -72,25 +72,32 @@
           end
         end
 
-        context 'when submitting an subscription that fails' do
+        context 'when submitting a subscription that fails' do
+          # Generating a genuine failure by violating uniqueness constraints
+          # in the CMR.
+          let(:name2) { 'Exciting Subscription with Important Data4' }
           before do
-            error_response_body = '{"errors":["some error message"]}'
-            error_response = cmr_fail_response(JSON.parse(error_response_body))
-            allow_any_instance_of(Cmr::CmrClient).to receive(:create_subscription).and_return(error_response)
+            @ingest_response, _concept_response = cmr_client.ingest_subscription({ 'Name' => name2, 'Query' => query, 'CollectionConceptId' => collection_concept_id, 'SubscriberId' => 'fake', 'EmailAddress' => 'fake@fake.fake'}.to_json, 'MMT_2', 'test_native_id', 'token')
```

Comment:
We don't need to do this now as there aren't many, but we will probably want to create a helper method to ingest these, like we do for the metadata records

Reply:
Adding to the list of nice to haves: rake command to set permissions